internal/config/config.go

```diff
@@ -14,6 +14,7 @@ import (
 const (
 	joyrcFile     = ".joyrc"
 	joyDefaultDir = ".joy"
+	joyCacheDir   = ".joycache"
```

Comment:
I don't think we should clobber the home dir more than we already do.

We should read `XDG_CACHE_HOME` and default to `~/.cache/joy` if it is not defined.

On macOS it would be more correct to be `~/Library/Caches/joy`, but `~/.cache` should be  good enough.